src/geom/elem.C

```diff
@@ -2442,12 +2442,19 @@ std::string Elem::get_info () const
 
   oss << "  Elem Information"                                      << '\n'
       << "   id()=";
-
   if (this->valid_id())
     oss << this->id();
   else
     oss << "invalid";
 
+#ifdef LIBMESH_ENABLE_UNIQUE_ID
+  oss << ", unique_id()=";
+  if (this->valid_unique_id())
+    oss << this->unique_id();
+  else
+    oss << "invalid";
+#endif
+
```

Comment:
Is it really worth it?  It's just an informational message... hardly seems like it's worth the trouble in rebasing and splitting that commit...